src/static/css/video-player.less

```diff
@@ -0,0 +1,225 @@
+
+/* ==========================================================================
+   cfgov-refresh
+   video-player
+   ========================================================================== */
+
+/* topdoc
+  name: Video Player
+  family: cfgov-video-player
+  patterns:
+      notes:
+        - "For the markup please see macros/video-player.html."
+  tags:
+    - cfgov-video-player
+*/
+
+// Variable used to set the size of the play button
+@play-btn-size__px: 73px;
+
+// Variable used to set the play button image
+@play-btn-url: '/static/img/video-play-btn.png';
+
+
+// Variable used to help center the video container
+@share-width__px: 130px;
+
+// Variables used for gutters
+@gutter__em: unit(@grid_gutter-width  / @base-font-size-px, em);
+@gutter-half__em: @gutter__em / 2;
+
+.video-player {
+    overflow: hidden;
+    background: @white;
+}
+
+.video-player.video-playing {
+    background-color: #000;
+
+    .hide-on_video-playing {
+        display: none;
+    }
+
+    .show-on_video-playing {
+        display: block;
+    }
+
+    .respond-to-max(@mobile-max, {
+        margin-top: 0;
+    });
+}
+
+// Video image container related styles
+
+.video-player_image-container {
+    position: relative;
+    background-color: #000;
+    text-align: center;
+
+    .respond-to-max(@mobile-max, {
+        .u-flexible-container();
+    });
+}
+
+.video-player_play-btn {
+    position: absolute;
+    height: @play-btn-size__px;
+    width: @play-btn-size__px;
+    background: url( @play-btn-url ) no-repeat;
+    z-index: 10;
+
+    &__center {
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%);
+    }
+
+    &__bottomRight {
+        bottom: @gutter-half__em;
+        right: @gutter-half__em;
+    }
+}
+
+.lt-ie9 {
+    .video-player_play-btn {
+        margin-left: -1 * @play-btn-size__px;
+    }
+}
+
+.video-player_video-container {
+    display: none;
+    margin: auto;
+}
+
+.video-player_video-container__flexible {
+    .video-player_iframe-container {
+        .u-flexible-container();
+    }
+
+    .video-player_video {
+        .u-flexible-container_inner();
+    }
+}
+
+.video-player_video-actions-container {
+    .grid_wrapper();
+
+    margin: 1em 0;
+    color: @white;
+
+    .share_heading {
+        margin-top: 0;
+        margin-right: 0;
+        line-height: 1;
+        white-space: nowrap;
+    }
+}
+
+.video-player_share-actions {
+    // Adding white background to the share icons
+    // without overlap.
+    .cf-icon {
+        height: 20px;
+        width: 22px;
+        background: @white;
+
+        &:before {
+            position: relative;
+            top: -3px;
+        }
+    }
+
+    .share-icon {
+        color: @pacific;
+    }
+
+    .respond-to-min(@tablet-min, {
+        text-align: right;
+    });
+}
+
+.video-player_share-actions, .video-player_video-actions {
+     margin-top: @gutter-half__em;
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/38a787781f8ef2dcd1d5aaebedd239e3a388d9cd